incoming-sms.php

```diff
@@ -8,20 +8,22 @@
 
 // Configure HTTP basic authorization: fc
 $config = FreeClimb\Api\Configuration::getDefaultConfiguration()
-              ->setUsername($_ENV['ACCOUNT_ID'])
-              ->setPassword($_ENV['API_KEY']);
+    ->setUsername($_ENV['ACCOUNT_ID'])
+    ->setPassword($_ENV['API_KEY']);
 
 $apiInstance = new FreeClimb\Api\Api\DefaultApi(
-    // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
-    // This is optional, `GuzzleHttp\Client` will be used as default.
+        // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
+        // This is optional, `GuzzleHttp\Client` will be used as default.
     new GuzzleHttp\Client(),
     $config
 );
 
 $account_id = $_ENV['ACCOUNT_ID']; // string | ID of the account
 $data = array(
-    'from' => '', //FC Number
-    'to' => '', //Verified Number
+    'from' => $_ENV['FROM'],
+    //FC Number
+    'to' => $_ENV['TO'],
```

Comment:
in the README it says to set the values in this file:

![image](https://github.com/FreeClimbAPI/PHP-SMS-Quickstart/assets/31379789/9d82261f-eb45-4a4a-948c-1589b7919c34)

for consistency, we need to either:
- revert back to local variables instead of env vars, or
- remove step 3 from the README and add the table info about these variables to the table about env vars in step 2

i think all other sms quickstarts use local variables for the 'to' and 'from' values, so we may want to stick with that convention

Reply:
but from the Java SDK standpoint, we grab from the environment variables and assign them to the local 'to' and 'from' variables in the code.

I am going to remove the step 3 and add table to the env vars table in step 2, if that is alright?